third_party/libtock-drivers/src/nfc.rs

```diff
@@ -23,25 +24,26 @@ mod allow_nr {
     pub const RECEIVE: usize = 2;
 }
 
+pub enum TransOrRecvStatus {
+    Error,
+    InvalidBuffer,
+    OOM,
+    Success,
+}
```

Comment:
As suggested below, you could either reuse an existing error code `enum` or remove `Success` and use it inside of `Result`s.